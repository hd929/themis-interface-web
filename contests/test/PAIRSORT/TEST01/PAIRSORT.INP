13
-42 -76
47 43
12 7
-62 31
7 64
42 -92
-89 60
45 41
4 54
-41 40
20 -24
88 42
0 12
