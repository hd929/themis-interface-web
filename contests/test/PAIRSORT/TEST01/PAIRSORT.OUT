0 12
12 7
20 -24
4 54
-41 40
45 41
47 43
7 64
-62 31
-42 -76
88 42
42 -92
-89 60
